common/djangoapps/util/views.py

```diff
@@ -308,7 +322,7 @@ def _record_feedback_in_zendesk(
 
 
 def _record_feedback_in_datadog(tags):
-    datadog_tags = [u"{k}:{v}".format(k=k, v=v) for k, v in tags.items()]
+    datadog_tags = [u"{k}:{v}".format(k=k, v=v) for k, v in sorted(tags.items())]
```

Comment:
You don't need to do this, just use 
assertItemsEqual in python2.7 and assertCountEqual with python3.5 (compare iterables while disregarding order)